local-tests/chainrunner/utils.py

```diff
@@ -0,0 +1,38 @@
+import os.path as op
+import re
+import subprocess
+from collections import OrderedDict
+
+
+def Generate_keys(binary, phrases):
```

Comment:
checkout the [bip-utils](https://pypi.org/project/bip-utils/) package, which might be useful here .

Sample usage:
```py

from bip_utils import SubstrateBip39SeedGenerator, SubstrateCoins, Substrate


phrase = "bottom drive obey lake curtain smoke basket hold race lonely fit walk"
seed_bytes = SubstrateBip39SeedGenerator(phrase).Generate()
substrate_ctx = Substrate.FromSeedAndPath(seed, '//0', SubstrateCoins.GENERIC)
print(substrate_ctx.PublicKey().ToAddress())
```

Reply:
This is awesome, thanks! :)